src/mmrelay/plugins/health_plugin.py

```
@@ -123,7 +133,7 @@ async def handle_meshtastic_message(
         return False
 
     async def handle_room_message(
-        self, room: MatrixRoom, event: RoomMessageText, full_message: str
+        self, room: MatrixRoom, event: dict[str, Any], full_message: str
```

Ruff (0.14.10)

Unused method argument: `full_message`

(ARG002)